src/test/java/com/ably/kafka/connect/DefaultChannelSinkMappingTest.java

```diff
@@ -0,0 +1,40 @@
+package com.ably.kafka.connect;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DefaultChannelSinkMappingTest {
+    private DefaultChannelSinkMapping defaultChannelSinkMapping;
+
+    //dependencies
+    private static final String STATIC_CHANNEL_NAME = "sink-channel";
```

Comment:
This static channel name is supposed to represent the channel name for which we use the 'static' mode, it doesn't reflect the static field that it's declared with. I agree it would have been superfluous 😄

Reply:
ok, fair point. Thanks for explaining.